src/components/lib/jumpTo.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <el-card>
+    <i class="el-icon-s-promotion" />
+    Jump to
+    <el-input
+      placeholder="PID"
+      v-model="jumpToProblem"
+      class="input-with-select" 
+      style="margin-top: 9px;"
+    >
+      <template slot="prepend">#.</template>
+      <el-button slot="append" icon="el-icon-top-right" @click="jump"></el-button>
+    </el-input>
+  </el-card>
+</template>
+
+<script>
+export default {
+  name: 'jumpTo',
```

Comment:
we use large-camel-case like `LargeCamelCase` or `JumpToProblem` for all components